scripts/gencert.service

```diff
@@ -1,12 +1,13 @@
 [Unit]
-Description=Generate xapi ssl certificate
+Description=Generate TLS certificates for xapi
 Requires=forkexecd.service
 After=forkexecd.service
 
 [Service]
 User=root
 Type=oneshot
-ExecStart=/opt/xensource/libexec/gencert /etc/xensource/xapi-ssl.pem
+ExecStart=/bin/sh -c '/opt/xensource/libexec/gencert /etc/xensource/xapi-ssl.pem default \
+                   && /opt/xensource/libexec/gencert /etc/xensource/xapi-pool-tls.pem xapi:pool'
```

Comment:
I think gencert.service runs after a toolstack restart, so this should generate 'xapi-pool-tls.pem' after an upgrade, as expected :+1:

Reply:
Note that the command exits with 0 if the cert already exists, which means that xapi-pool-tls.pem will be generated even if xapi-ssl.pem exists.

Reply:
Yes, I made sure of that :)